cmake/scripts/darwin_embedded/Install.cmake

```diff
@@ -51,6 +51,14 @@ add_custom_command(TARGET ${APP_NAME_LC} POST_BUILD
     COMMAND "ACTION=build"
             ${CMAKE_SOURCE_DIR}/tools/darwin/Support/CopyRootFiles-darwin_embedded.command
     COMMAND "XBMC_DEPENDS=${DEPENDS_PATH}"
+            "TARGET_BUILD_DIR=$<TARGET_FILE_DIR:${APP_NAME_LC}>/.."
+            "TARGET_NAME=${APP_NAME}.app"
+            "APP_NAME=${APP_NAME}"
+            "PRODUCT_NAME=${APP_NAME}"
+            "FULL_PRODUCT_NAME=${APP_NAME}.app"
+            "WRAPPER_EXTENSION=app"
+            "SRCROOT=${CMAKE_BINARY_DIR}"
```

Comment:
looks like a rebase error

Reply:
Why? PYTHON_VERSION now needs to be set so I took the version that contained that. Do you know what it should be?

Reply:
Yes it needs `PYTHON_VERSION`, but that's not in line 54 to 60.
It's in line 61 which is correct and I didn't include in this multi line comment.

Reply:
Okay, but they were a package. Do you want it with just line 61?

Reply:
Have a look at the commit history of this file, this lines where recently removed by intention. You pulled them back while rebasing.

Reply:
okay. When I was resolving the commit conflict I had the choice between all of the lines, or none of the lines. I took the one that included them all because otherwise PYTHON_VERSION wouldn't have been defined, which I saw was needed. That's why they're all there. If you say only PYTHON_VERSION is needed, I'll fix it, but that's why I resolved it that way,

Reply:
Then you are using the wrong tool for rebasing if it only gives you the choice to pick one or the other version and not to interactively fix it.

Reply:
No. You misunderstood me. I could have manually merged them but assuming the state of the code that accompanied the PYTHON_VERSION addition was the correct state was a natural, even if incorrect, assumption.

Reply:
Ah okay.

Please squash the fix commits into the corresponding commits, to not have the wrong changes in the history.